### ISSUE TITLE
unocss lsp throws: -32601 unhandled method textdocument/documenthighlight

### ISSUE THREAD

**Issue by rajshekardev** [xna00/unocss-language-server#7]
Using the unocss-ls throws the error in Astrovim. 

1. Minimal neovim lua conf with Lazy.nvim & mason. 
2. NvChad stock conf from thier site.
3. AstroNvim stock config

other LSP installed include Volar for vue, Typescript, lua,

In both cases 1 & 2 the Unocss doesnt work at all. When i use 3 it throws the error: -32601 unhandled method textdocument/documenthighlight

In all three cases above when i open a vue, nuxtJS, NextJS projects with unocss config file i get the issue. 
 I tried multiple configs in different projects. 

Even starting a new Nuxt project and adding unocss causes the issue.

relatively new to NeoVim and modal editors. so i dont know what else i can add here.

**Comment by getcisher** [comment]
Same here. I using Nuxt Project + unocss with minimum config in AstroNvim.
Did the unocss-lsp config conflict with package in mason?

**Comment by Readpato** [comment]
Having the same problem over here. Problem happening with AstroNvim and a Nuxt project. Would love to help on fixing this but wouldn't know where to start!

**Comment by fr0stf0x** [comment]
you can put this setting under "unocss" lspconfig

```
-- user/init.lua
{
	-- other settings
	lsp = {
		config = {
			-- others lsp
			unocss = {
		        handlers = {
		          ["textDocument/documentHighlight"] = function() end,
		        },
			}
	},
}
```

**Comment by xna00** [comment]
Fixed in v0.1.3.
